t/venom/test_suites/common/restart_pfdns_service.yml

```diff
@@ -1,17 +1,6 @@
 name: Restart pfdns service
```

Comment:
We have two files which do same thing:
- restart_pfdns.yml
- restart_pfdns_service.yml

Reply:
I removed restart_pfdns.yml to be consistent with current naming convention.